### PR TITLE
release notes for 0.6.0-RC1

### PR DESCRIPTION
**Pull request by tyrasd**
let's start with a RC, so we can already start testing the 0.6 stuff.

RC2 we then also have #283 and #241.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,42 @@
...
+* better handling of OSM multipolygons with touching inner rings. This improves performance considerably in some cases (especially large complex multipolygons). #249
 * improve performance of `aggregateByGeometry` queries. #272
+* improve performance of geometry-building of relations with a huge number members. #287
+
+### bugfixes and other changes
+
+* compatibility fix to allow building of javadoc under Java 11
+* fix bug where in some cases, instead of an OSHDBTimeoutException an IniteException was thrown. #258
+* various code style and code quality improvements
+* the OSHDB is now published on Zenodo for easier citation using the DOI `10.5281/zenodo.4146991`
```
Comment (joker234) [inline CHANGELOG.md:38]: I would use the general DOI 4146990 here. Should we link the DOI (using https://dx.doi.org/10.5281/zenodo.4146990) somehow? Or do we prevent that in Changelogs?
```suggestion
* the OSHDB is now published on Zenodo for easier citation using the DOI `10.5281/zenodo.4146990`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,42 @@
 Changelog
 =========
 
-## 0.6.0 SNAPSHOT (current master)
+## 0.6.0-RC1
+
+### breaking changes
 
 * reorganize maven packages: rename group parent to ohsome-parent (version bump to 2.3), rename local parent to oshdb-parent, and change groupId to org.heigit.ohsome
```
Comment (joker234) [inline CHANGELOG.md:8]: do we have a PR number for this change?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,16 +1,42 @@
...
-* compatibility fix to allow building of javadoc under Java 11
-* fix bug where in some cases, instead of an OSHDBTimeoutException an IniteException was thrown. #258
+
+When switching to the OSHDB version 0.6 you need to adapt your `pom.xml` to the new groupId, e.g.:
```
Comment (joker234) [inline CHANGELOG.md:10]: Should we mention that the `import`s in each file need to be changed?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,16 +1,42 @@
...
+```xml
+<dependency>
+  <groupId>org.heigit.ohsome</groupId>
+  <artifactId>oshdb-api</artifactId>
+  <version>0.6.0</version>
+</dependency>
+```
+
+* Timestamp parser class renamed to `IsoDateTimeParser` from `ISODateTimeParser` and adjust how input timestamps (e.g. in `MapReducer.timestamps()`) are handled: only the UTC time zone identifier `Z` is supported. #265
+* Removed `UNKOWN` from the `OSMType` enumeration class.
```
Comment (joker234) [inline CHANGELOG.md:20]: do we have a PR number for this change?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,16 +1,42 @@
 Changelog
 =========
 
-## 0.6.0 SNAPSHOT (current master)
+## 0.6.0-RC1
```
Comment (joker234) [inline CHANGELOG.md:4]: General question: do we keep the RC-Changelog or do we rename `0.6.0-RC1` later to `0.6.0` (and eventually add further entries).

File: CHANGELOG.md (hunk 2 above)
Comment (tyrasd) [reply CHANGELOG.md:8]: yeah, #234 and #257 

btw, I see now that _version bump to 2.3_ is also not 100% correct anymore.

File: CHANGELOG.md (hunk 3 above)
Comment (tyrasd) [reply CHANGELOG.md:10]: I would add that to the changelog with #241

File: CHANGELOG.md (hunk 5 above)
Comment (tyrasd) [reply CHANGELOG.md:4]: I would rename it later to `0.6.0`

File: CHANGELOG.md (hunk 1 above)
Comment (joker234) [reply CHANGELOG.md:38]: The link is already in the README.

**Review by joker234 — APPROVED** [review]

**Pull request merged**